core/src/authorization/authorization.service.ts

```diff
@@ -45,4 +46,61 @@ export class AuthorizationService {
 
         return Array.from(actions);
     }
+
+    async getUserActions(userId: number, organizationId: number): Promise<string[]> {
```

Comment:
We almost certainly want to cache this somehow, huge overhead being added here because we are pulling a complete list of actions every time we want to check one.